### PR TITLE
Replace the header image with ALab.

### PR DESCRIPTION
**Pull request by vickycl**
## Purpose of this PR

**Ticket/Jira #:** https://jira.unity3d.com/browse/USDK-260

This PR removes the Kitchen image which we don't have permission for and replaces it with ALab. I also attempted to add a third party notice, lmk if it's good or bad.

## Testing

**Functional Testing status:**

**Performance Testing status:**

## Overall Product Risks

**Complexity:**

**Halo Effect:**

## Additional information

**Note to reviewers:**

**Reminder:**

- [ ] Add entry in CHANGELOG.md _(if applicable)_

### CODE REVIEW

File: package/com.unity.formats.usd/Third Party Notices.md [hunk 1]
```diff
@@ -841,4 +841,22 @@ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
...
+The assets and their output may only be referred to as the Asset Name listed above, and your use of the Asset Name shall be solely to identify the digital assets. Other than as expressly permitted by this License, you may NOT use any trade names, trademarks, service marks, or product names of the copyright holder for any purpose.
+DISCLAIMER: THESE DIGITAL ASSETS ARE PROVIDED BY THE COPYRIGHT HOLDER “AS IS” AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE, ARE DISCLAIMED. IN NO EVENT SHALL COPYRIGHT HOLDER BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THESE DIGITAL ASSETS, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```
Comment (vickycl) [inline package/com.unity.formats.usd/Third Party Notices.md:863]: @michaeljblain I'm not sure if this is right or even necessary, could you check it over please?

Comment (michaeljblain) [reply package/com.unity.formats.usd/Third Party Notices.md:863]: Yes we do need to include it. I'll take a look if anything else is required.

**Review by michaeljblain — APPROVED** [review]
Looks good. Just need to check ThirdParty aspect.

File: package/com.unity.formats.usd/Third Party Notices.md (hunk 1 above)
Comment (michaeljblain) [reply package/com.unity.formats.usd/Third Party Notices.md:863]: Looks good.

**Pull request merged**